lib/saml/user_attributes/ssoe.rb

```diff
@@ -54,6 +58,16 @@ def icn
         mvi_ids[:icn]
       end
 
+      def address
+        {
```

Comment:
Prob need to change this to: `OpenStruct.new({
           street: safe_attr('va_eauth_street'),
           city: safe_attr('va_eauth_city'),
           state: safe_attr('va_eauth_state'),
           country: safe_attr('va_eauth_country'),
           postal_code: zip
         })`